website/docs/docs/building-a-dbt-project/building-models.md

```diff
@@ -3,18 +3,197 @@ title: "dbt Models"
+```
+
+</File>
+
+When you execute `dbt run`, dbt will build this as a view named `customers` in your target schema:
+
+```sql
+create view dbt_alice.customers as (
+    with customer_orders as (
+        select
+            customer_id,
+            min(order_date) as first_order_date,
+            max(order_date) as most_recent_order_date,
+            count(order_id) as number_of_orders
+
+        from jaffle_shop.orders
+
+        group by 1
+    )
+
+    select
+        customers.customer_id,
+        customers.first_name,
+        customers.last_name,
+        customer_orders.first_order_date,
+        customer_orders.most_recent_order_date,
+        coalesce(customer_orders.number_of_orders, 0) as number_of_orders
+
+    from jaffle_shop.customers
+
+    left join customer_orders using (customer_id)
+)
+```
+
+Why a _view_ named `dbt_alice.customers`? By default dbt will:
+* create models as views
+* build models in a target schema you define
+* use your file name as the view or table name in the database
+
+You can use _configurations_ to change any of these behaviors — more on that below.
+
+### FAQs
+<FAQ src="checking-logs" />
+<FAQ src="create-a-schema" />
+<FAQ src="run-downtime" />
+<FAQ src="sql-errors" />
+<FAQ src="sql-dialect" />
+
+## Configuring models
+Configurations are "model settings"  that can be set in your `dbt_project.yml` file, _and_ in your model file using a `config` block. Some example configurations include:
+* Change the [materialization](materializations) that a model uses — a materialization determines the SQL that dbt uses to create the model in your warehouse.
+* Build models into separate [schemas](using-custom-schemas).
+* Apply [tags](resource-configs/tags) to a model.
+
+Here's an example of model configurations:
+
+<File name='dbt_project.yml'>
+
+```yaml
+name: jaffle_shop
+
+models:
+  jaffle_shop: # this matches the `name:`` config
+    materialized: view # this applies to all models in the current project
+      marts:
+        materialized: table # this applies to all models in the `marts/` directory
+        marketing:
+          schema: marketing # this applies to all models in the `marts/marketing/`` directory
+
+```
+
+</File>
+
+
+<File name='models/customers.sql'>
+
+```sql
+
+{{ config(
+    materialized="view",
+    schema="marketing"
+) }}
+
+with customer_orders as ...
+
+```
+
+</File>
+
+Importantly, configurations are applied hierarchically — a configuration applied to a subdirectory will override any general configurations.
+
+You can learn more about configurations in the [reference docs](model-configs).
+
+### FAQs
+<FAQ src="available-materializations" />
+<FAQ src="available-configurations" />
+
+
+## Building dependencies between models
+By using the [`ref` function](ref) in the place of table names in a query, you can build dependencies between models. Use the name of another model as the argument for `ref`.
+
+
+<File name='models/customers.sql'>
```

Comment:
Do you think it would be compelling to tabify this and show:
 - source code
 - dev env
 - prod env

with different value interpolated in each tab?